_data/blogs.yml

```diff
@@ -197,3 +197,8 @@
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
   author_name: Saarthak Chaturvedi
+
+- url: https://limitless497584285.wordpress.com/2017/12/27/coding-my-joy/
+  title: CODING- MY JOY
+  img: pmslife.jpg
```

Comment:
Please upload the image with this file name of 240x240.

Reply:
Here's the link
https://github.com/fossasia/gci17.fossasia.org/pull/573/files/63dddbfb9662c790e2cb0b6c7e1daa0f74c5a80b